WalletWasabi/WabiSabi/Backend/Models/WabiSabiProtocolErrorCode.cs

```diff
@@ -33,7 +33,7 @@ public enum WabiSabiProtocolErrorCode
 	DeltaNotZero,
 	WrongNumberOfCreds,
 	CryptoException,
-	AliceAlreadySignalled,
+	AliceAlreadySignaled,
```

Comment:
Is this change correct?

Reply:
I would be very hesitant about this change.

Reply:
Revert this, might breaks the compatibility.